src/vcpkg/remove.cpp

```diff
@@ -305,8 +302,12 @@ namespace vcpkg::Remove
             Checks::exit_success(VCPKG_LINE_INFO);
         }
 
-        for (const RemovePlanAction& action : remove_plan)
+        // note that we try to "remove" things that aren't installed to trigger purge actions
+        for (std::size_t idx = 0; idx < remove_plan.size(); ++idx)
         {
+            const RemovePlanAction& action = remove_plan[idx];
+            msg::println(msg::format(msgRemovingPackage, msg::spec = action.spec)
+                             .append_fmt_raw(" ({}/{})...", idx + 1, remove_plan.size()));
```

Comment:
I see you added `append_fmt_raw` now; is this (x/y) notation common enough that we should add an explicit fn for it? (There are at least 2-3 such examples touched in this PR alone but I don't have a barometer on how many others there are)

Reply:
These 2-3 are the only ones I've touched recently. I think that refactor can be done separately if/when it's deemed needed.